gluster/cli/volume.py

```diff
@@ -226,7 +226,7 @@ def optreset(volname, opt=None, force=False):
     return volume_execute(cmd)
 
 
-def vollist():
+def list():
```

Comment:
Named as vollist to avoid collision with Python list

    from gluster.cli.volume import list

vs 

    from gluster.cli.volume import vollist